debs/nginx-ldap-auth/debian/control

```diff
@@ -0,0 +1,13 @@
+Source: nginx
+Section: web
+Priority: optional
+Maintainer: Osirium <support@osirium.com>
+Build-Depends: debhelper (>= 8.0.0), libssl-dev, libpcre++-dev
+Standards-Version: 3.9.4
+Homepage: https://www.osirium.com
+
+Package: nginx
```

Comment:
Given this is really a different package with different capabilities, IMO it should be called something else & either `Conflicts: nginx` or just install to a different path.
There's a risk here that a newer `nginx` from the official repos could get installed as part of an update and remove LDAP support, right?
(pinning would be another option, but less visible & less correct. Maybe we already do that though?)

Reply:
.deb package itself has a different version/name and its also located in different folder 
I must've missed changing the package bit, thank you for pointing this out